### DISCUSSION TITLE
Add multiple areas / masks

### DISCUSSION THREAD

**Discussion by Tihmann** [Motion-Project/motion#1483] [General]
Could you please add the possibility to add multiple areas or masks ? Each area / mask should trigger a different action. A mask that - just like the static mask to lower sensitivity - defines the area where a script will be executed. The code to read masks is there and the detection overhead is just one if-statement more per loop.

**Comment by tosiara** [comment]
So you want multiple masks and multiple events per image?

**Comment by Tihmann** [comment]
My camera is facing the hallway with a glass entrance door. I would like a set a mask that triggers a script when motion is detected in front of the glass entrance door, e.g. to turn on the lights in the hallway. I would then define a second mask which triggers an alarm when motion is detected inside the hallway itself, e.g. by an intruder.
